<launch>
	<!-- Run srs_interaction_primitives service server -->
	<node name="but_interaction_primitives_service_server" pkg="srs_interaction_primitives" type="but_interaction_primitives_service_server" />

	<!-- Run rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-f /world -d $(find srs_interaction_primitives)/config/interaction_primitives.vcg"/>
</launch>
